src/test/java/org/folio/spring/controller/TenantControllerTest.java

```diff
@@ -0,0 +1,107 @@
+package org.folio.spring.controller;
+
+import static org.folio.spring.integration.XOkapiHeaders.TENANT;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.zonky.test.db.AutoConfigureEmbeddedDatabase;
+import javax.sql.DataSource;
+import lombok.SneakyThrows;
+import org.folio.tenant.domain.dto.TenantAttributes;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.flyway.FlywayAutoConfiguration;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.web.servlet.MockMvc;
+
+@SpringBootTest
+@AutoConfigureEmbeddedDatabase(beanName = "dataSource")
+@EnableAutoConfiguration(exclude = FlywayAutoConfiguration.class)
+@AutoConfigureMockMvc
+class TenantControllerTest {
```

Comment:
I assume it's an integration test, right? Would it be better to name it respectively? 
![image](https://user-images.githubusercontent.com/42244720/106487001-e72b4780-64ba-11eb-8a35-3f3830fa8879.png)

Reply:
Yes, renamed to `TenantControllerIT`